handsontable/src/core.js

```diff
@@ -3369,7 +3371,7 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     return metaManager.getCellMeta(physicalRow, physicalColumn, {
       visualRow: row,
       visualColumn: column,
-    });
+    }, options);
```

Comment:
The 3rd argument of the `getCellMeta` is already an option designed for extending the meta. No need to create 4th one.
``` js
return metaManager.getCellMeta(physicalRow, physicalColumn, {
      visualRow: row,
      visualColumn: column,
      ...options,
    });
```

Reply:
Changed